crates/node/src/network_server/handler/cluster_ctrl.rs

```diff
@@ -34,7 +35,9 @@ impl ClusterCtrlSvc for ClusterCtrlSvcHandler {
         &self,
         request: Request<AttachmentRequest>,
     ) -> Result<Response<AttachmentResponse>, Status> {
-        let node_id = request.into_inner().node_id.expect("node id must be set");
+        let node_id = NodeId::from(request.into_inner().node_id.expect("node id must be set"))
+            .as_generational()
+            .expect("generational id");
```

Comment:
For the future implementation of this: It doesn't feel great to panic on malformed requests, but it's okay to keep this since it's a stub anyway.

Reply:
True. Let me actually remove this source of panic since there is no need for it even as part of the stub.